### PR TITLE
Include optional question to look at an example of a bad training curve

### LINKED ISSUE #278: Improve solution to "The Training Curve" exercise
(With apologies if there is an issue already open about this.)

In my view, the solution provided for [the _The Training Curve_ exercise](https://carpentries-incubator.github.io/deep-learning-intro/02-keras/index.html#the-training-curve) could be improved. At the moment, the exercise and the text before it talk about the potential value of plotting out the loss curve, as a way to evaluate the quality of the training configuration. However, I think it leaves unanswered the question many learners would have at this point - "what does a poorly-configured training process look like and how can I fix it?" I think it could be improved by adding an example of a loss plot for a poorly-configured training process, and by explaining what "jitter" in the curve might indicate/how that could be addressed, etc. Perhaps you could even set another exercise that requires them to generate a loss curve for a poorly-configured training process? (Or that might take too long to achieve?)

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes #278

### CODE REVIEW

File: episodes/2-keras.Rmd [hunk 1]
```diff
@@ -489,11 +489,30 @@ Looking at the training curve we have just made.
    * Does the graph look very jittery?
 2. Do you think the resulting trained network will work well on the test set?
 
+When the training process does not go well:
+
+3. (optional) Something went wrong here during training. What could be the problem, and how do you see that in the training curve?
+Also compare the range on the y-axis with the previous training curve.
+![](../fig/02_bad_training_history_1.png){alt='Very jittery training curve. The loss does not really decrease'}
```
Comment (dsmits) [inline episodes/2-keras.Rmd:496]: I dunno, I think the loss definitely does decrease, not as much though. Would it help to run this same thing with more iterations so you can see that it stagnates for a longer time?

**Review by dsmits — CHANGES_REQUESTED** [review]

File: episodes/2-keras.Rmd (hunk 1 above)
Comment (tobyhodges) [reply episodes/2-keras.Rmd:496]: I agree that the trend of the curve shows some decrease with epochs. I think you could resolve this by adjusting the alt-text to describe the range on the y-axis: this would also be consistent with the sentence above ("Also compare the range on the y-axis with the previous training curve.").

Comment (svenvanderburg) [reply episodes/2-keras.Rmd]: Good points! I resolved it by having a much more elaborate alt text, indeed referring to the fact that the y-axis range is different.
